logos.json

```diff
@@ -17,12 +17,11 @@
 		]
 	},
 	{
-		"title": "Mainstreet Bank",
-		"filename": "mainstreet_bank",
-		"url": "http://mainstreetcapltd.com/",
+		"title": "Cal_State_Logos",
+		"filename": "Cal_State_Logos",
+		"url": "http://www.fullerton.edu/",
 		"category": [
-			"Banking",
-			"Financial Services"
+			"Education",
```

Comment:
Hey @janessa17 
It looks like you're overwriting information that was there previously instead of adding to the end of the existing `logos.json` file.
Please correct this, ensure everything works as should on your local and then push a new commit with the changes to this PR.

You can refer to the [contribution document](https://github.com/PaystackHQ/nigerialogos/blob/master/contributing.md) for more help with adding logos.